tests/test_application.py

```diff
@@ -384,23 +394,18 @@ def test_builder(self, app):
         builder_1.token(app.bot.token)
         builder_2.token(app.bot.token)
 
-    @pytest.mark.parametrize("job_queue", (True, False))
-    async def test_start_stop_processing_updates(self, bot, job_queue):
+    # @pytest.mark.parametrize("job_queue", (True, False))
```

Comment:
We'll still want to test both cases. The test probably failed due to the new warnings being issued, right? You can suppress that by adding a decorator

```
@pytest.mark.filterwarnings("ignore::telegram.warnings.PTBUserWarning")
```

Same for `TestApplicationBuilder.test_no_job_queue` btw :)